docs/getting_started.rst

```diff
@@ -209,14 +207,20 @@ For creating a message with data, first we must to decide which stream we want t
 .. code-block:: C
 
     HelloWorld topic = {count++, "Hello DDS world!"};
-    (void) mr_write_HelloWorld_topic(&session, reliable_out, datawriter_id, &topic);
+
+    MicroBuffer mb;
+    uint32_t topic_size = HelloWorld_size_of_topic(&topic, 0);
+    (void) mr_prepare_output_stream(&session, reliable_out, datawriter_id, &mb, topic_size);
+    (void) HelloWorld_serialize_topic(&mb, &topic);
 
     mr_run_session_until_confirmed_delivery(&session, 1000);
 
-``mr_write_HelloWorld_topic`` function is automatically generated by :ref:`micrortpsgen_label` from the IDL.
-This function serializes the topic into stream.
-If the stream is available and the topic fix into it, a valid *request id* is returned.
-``datawriter_id`` correspond to the data writer entity used for sending the data.
+``HelloWorld_size_of_topic`` and ``HelloWorld_serialize_topic`` functions are automatically generated by :ref:`micrortpsgen_label` from the IDL.
+The function ``mr_prepare_output_stream`` request a writing for a topic of ``topic_size`` size into the reliable stream represented by ``reliable_out``,
+with a ``datawriter_id`` (correspond to the data writer entity used for sending the data in the `DDS World`).
+If the stream is available and the topic fit into it, the function will initialize the ``MicroBuffer`` structure ``mb``.
+Once the ``MicroBuffer`` is prepared, the topic can be serialized into it.
+We have careless about ``mr_prepare_output_stream`` return value because the serialization only will occurs if the ``MicroBuffer` is valid``
```

Comment:
"We are careless" and "the serialization will only occur"

Reply:
done